### DISCUSSION TITLE
Requiring code reviews to merge

### DISCUSSION THREAD

**Discussion by objecttothis** [opensourcepos/opensourcepos#3672] [Team Posts]
@jekkos @daN4cat what do you think about requiring at least one code review to merge a PR?
https://help.github.com/en/github/administering-a-repository/enabling-required-reviews-for-pull-requests

I don't know that it's an issue now, but it could prevent having to revert merged PR because bugs weere found later when they could have been spotted had there been a requirement for at least one code review.

**Comment by jekkos** [comment]
Good idea, I think that once someone makes a PR, we should indeed at least have on other person that reviews it. 

The required action depends on what the content is. 
* If it's a simple bug fix then the PR is probably made to have someone else validate if it works.
* If it's a new feature, then the focus is more on code review (but probably also testing)

**Comment by objecttothis** [comment]
Can you make the changes to the restrictions?  I don't think I have permissions to make those changes to the repository.

**Comment by jekkos** [comment]
Ok I have enabled the rule for now. It's nice to see that github is improving functionality even after being acquired by M$.

**Comment by SteveIreland** [comment]
I won't be impacted.   @daN4cat ALWAYS reviews my code (at least it seems that way).  :-)
